smac/optimizer/pSMAC.py

```diff
@@ -47,6 +47,7 @@ def read(run_history: RunHistory,
             if match:
                 runhistory_file = os.path.join(output_directory,
                                                file_in_output_directory)
+                print(runhistory_file)
```

Comment:
We shouldn't have any print statements in our code.

Reply:
Sorry, my mistake. fixed now.